r/DESCRIPTION

```diff
@@ -0,0 +1,22 @@
+Package: rrrow
```

Comment:
I don't love the name. We should solicit some more feedback about what to call the R package (simply "arrow" seems good to me)

Reply:
As a R user in the peanut gallery, I think "arrow" seems great. Especially if there isn't a namespace collision on CRAN or bioconductor (I didn't notice any with a quick look). For example {tensorflow} and {docker} carry directly over from Python, so it makes them easy to remember.

Reply:
Sure. I've changed it to arrow.